diff --git a/arch/x86/platform/intel-mid/device_libs/platform_spidev.c b/arch/x86/platform/intel-mid/device_libs/platform_spidev.c
index 40827d64011..856411e9bca 100644
--- a/arch/x86/platform/intel-mid/device_libs/platform_spidev.c
+++ b/arch/x86/platform/intel-mid/device_libs/platform_spidev.c
@@ -20,7 +20,9 @@
 static void tng_ssp_spi_cs_control(u32 command);
 static void tng_ssp_spi_platform_pinmux(void);
 
-static int tng_ssp_spi2_FS_gpio = 111;
+//static int tng_ssp_spi2_FS_gpio = 111;
+// Ye-Sheng, 10/27/16
+static int tng_ssp_spi2_FS_gpio = 110;
 
 static struct intel_mid_ssp_spi_chip chip = {
 	.burst_size = DFLT_FIFO_BURST_SIZE,
@@ -33,26 +35,28 @@ static struct intel_mid_ssp_spi_chip chip = {
 
 static void tng_ssp_spi_cs_control(u32 command)
 {
-	gpio_set_value(tng_ssp_spi2_FS_gpio, (command != 0) ? 1 : 0);
+    // Ye-Sheng, 10/27/16
+	//gpio_set_value(tng_ssp_spi2_FS_gpio, (command != 0) ? 1 : 0);
 }
 
 static void tng_ssp_spi_platform_pinmux(void)
 {
-	int err;
-	int saved_muxing;
-	/* Request Chip Select gpios */
-	saved_muxing = gpio_get_alt(tng_ssp_spi2_FS_gpio);
+    // Ye-Sheng, 10/27/16
+	//int err;
+	//int saved_muxing;
+	///* Request Chip Select gpios */
+	//saved_muxing = gpio_get_alt(tng_ssp_spi2_FS_gpio);
 
-	lnw_gpio_set_alt(tng_ssp_spi2_FS_gpio, LNW_GPIO);
-	err = gpio_request_one(tng_ssp_spi2_FS_gpio,
-			GPIOF_DIR_OUT|GPIOF_INIT_HIGH, "Arduino Shield SS");
-	if (err) {
-		pr_err("%s: unable to get Chip Select GPIO,\
-				fallback to legacy CS mode \n", __func__);
-		lnw_gpio_set_alt(tng_ssp_spi2_FS_gpio, saved_muxing);
-		chip.cs_control = NULL;
-		chip.platform_pinmux = NULL;
-	}
+	//lnw_gpio_set_alt(tng_ssp_spi2_FS_gpio, LNW_GPIO);
+	//err = gpio_request_one(tng_ssp_spi2_FS_gpio,
+	//		GPIOF_DIR_OUT|GPIOF_INIT_HIGH, "Arduino Shield SS");
+	//if (err) {
+	//	pr_err("%s: unable to get Chip Select GPIO,\
+	//			fallback to legacy CS mode \n", __func__);
+	//	lnw_gpio_set_alt(tng_ssp_spi2_FS_gpio, saved_muxing);
+	//	chip.cs_control = NULL;
+	//	chip.platform_pinmux = NULL;
+	//}
 }
 
 void __init *spidev_platform_data(void *info)
diff --git a/arch/x86/platform/intel-mid/device_libs/platform_spidev.h b/arch/x86/platform/intel-mid/device_libs/platform_spidev.h
index a40ef8ff29a..6ac91cec45a 100644
--- a/arch/x86/platform/intel-mid/device_libs/platform_spidev.h
+++ b/arch/x86/platform/intel-mid/device_libs/platform_spidev.h
@@ -14,7 +14,9 @@
 
 /* REVERT ME workaround[MRFL] for invalid bus number in IAFW .25 */
 #define FORCE_SPI_BUS_NUM	5
-#define FORCE_CHIP_SELECT	1
+//#define FORCE_CHIP_SELECT	1
+// Ye-Sheng 10/26/16
+#define FORCE_CHIP_SELECT	0
 
 extern void *spidev_platform_data(void *info) __attribute__((weak));
 #endif
